packages/components/datagrid/src/js/datagrid.html

```diff
@@ -20,13 +20,17 @@
         <div class="oui-datagrid-responsive-container__overflow-container">
             <table class="oui-datagrid oui-datagrid_responsive">
                 <thead class="oui-datagrid__headers">
-                    <tr class="oui-datagrid__row">
+                    <tr class="oui-datagrid__row"
+                        ng-class="{'expandable': $ctrl.expandableRows}">
```

Comment:
The classname isn(t BEM compliant.

```suggestion
                        ng-class="{'oui-datagrid__row_expandable': $ctrl.expandableRows}">
```